wazo_agentd_client/commands/tests/test_agents.py

```diff
@@ -168,6 +168,20 @@ def test_status_all(self):
 
         self._assert_get_request(req, expected_url)
 
+    def test_user_agent_login_to_queue(self):
+        expected_url = f'{self.base_url}/users/me/agents/queues/1/login'
+
+        req = self.req_factory.user_agent_login_to_queue(1)
+
+        self._assert_put_request(req, expected_url)
+
+    def test_user_agent_logoff_to_queue(self):
```

Comment:
Just for consistency, I'd suggest to rename the test to `test_user_agent_logoff_to_queue`

```suggestion
    def test_user_agent_logoff_from_queue(self):
```